.editorconfig

```diff
@@ -20,11 +20,18 @@ csharp_style_expression_bodied_constructors = false:silent
 csharp_style_expression_bodied_operators = false:silent
 csharp_indent_labels = one_less_than_current
 
+[.editorconfig]
+end_of_line = lf
+
+# Git files
+[.gitignore]
+end_of_line = lf
+
 [*.sln]
 indent_style = tab
 
 # Xml project files
-[*.{config,csproj,njsproj,props,targets,vcxitems,vcxproj,vcxproj.filters}]
+[*.{config,csproj,njsproj,targets,vcxitems,vcxproj,vcxproj.filters}]
```

Comment:
This PR should not contain any changes to .editorconfig

Reply:
Every change in this PR is pertinent.

This PR was updated in a UNIX machine.

Without that change, the .gitignore update would cause an unwanted commit replacing all line endings on that file with CRLF as per current EditorConfig rules.
This is **exactly** what my autoCRLF PR aimed to address and fix for good.

I know working on this project in UNIX machines is not the norm, but it should be doable.
The changes to this file ensure there is no disruption in non-Windows machines.

Additionally, I removed a duplicate `props` entry in EditorConfig.

Kindly unblock this change.

Reply:
FFmpegInteropX cannot be built on Linux, so it's pointless to talk about this, but assuming that it would be - we still would not set any specific EOL style for .editorconfig and .gitignore because they would be checked out with CRLF on Windows and with LF on Linux, so there's no problem at all.

Reply:
I see you did read this, but you did not understand it.
I never talked about building this on Linux.
Some changes, like this one, can be done on any machine with a text editor. This should not be blocked nor purposefully unsupported.

If you carefully read the EditorConfig rules, they **already** set a specific EOL style for `.editorconfig` and `.gitignore`: CRLF via default rules.

This causes the files to add CRLF on save but, because UNIX does not use autoCRLF, it will check in the file as is.
Obviously, this won't be a problem on Windows if autoCRLF is on.

Setting these two files to LF affects you in no imaginable way and prevents anyone using non-Windows environments editing this project from running into issues.

Ergo, the change in this file is valid.

Reply:
No, this change is not valid, because the .editorconfig and .gitignore files are CRLF on Windows and when .editorconfig sets it to LF and, I get warnings about mixed line endings each time I edit the file.

> If you carefully read the EditorConfig rules, they already set a specific EOL style for .editorconfig and .gitignore: CRLF via default rules.

A valid change would be to unset EOL styles for those file types, but then it still would have to be in a separate PR, because that's totally unrelated.

Reply:
I think setting autocrlf to "input" will fix the problem your problem@JunielKatarn . Can you make a branch so we can test it out? Thanks.

Reply:
@brabebhin make a branch for what purpose, exactly? To test what?